app/forms/publish/user_form.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Publish
+  class UserForm < Form
+    FIELDS = %i[
+      first_name
+      last_name
+      email
+      id
+    ].freeze
+
+    attr_accessor(*FIELDS)
+
+    validates :first_name, presence: true
+    validates :last_name, presence: true
+    validates :email, presence: true, format: { with: /\A.*@.*\z/, message: :format }
+    validate :email_is_lowercase
+
+    def compute_fields
+      model.attributes.symbolize_keys.slice(*FIELDS).merge(new_attributes)
+    end
+
+    def fields_to_ignore_before_save
+      %i[authenticity_token code]
+    end
```

Comment:
Do you still need that?

Reply:
Nope 😶‍🌫️ 

https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/dd29ad8d9630d4a49e9c5a070aaa430b54401dd6